### DISCUSSION TITLE
UA标识导致操作出错

### DISCUSSION THREAD

**Discussion by CloudyEagle** [the1812/Bilibili-Evolved#3875] [综合讨论 / General]
![image](https://user-images.githubusercontent.com/58777085/207475110-c1b1c6c0-bcd1-4e2a-ae9a-296360babf8f.png)
我一直有开UA修改器，以模拟MacOS上的Safari浏览器。个人感觉看视频时会更好地调度显卡解码，但是近期发现开启后鼠标无法正常操作。
鼠标暂停本应该点击一下播放画面就好，现在需要鼠标双击才能暂停，仿佛是给了触摸屏的操作逻辑，并且也无法对弹幕做快捷操作。
上述现象均排除了其他脚本和插件的干扰，请问这是什么情况，在执意模拟此UA的情况下，怎样才能正常操作。
此外这是我在Win10上的Edge最新的稳定版、beta版和dev版发现，在要停止支持的Windows 7上无法复现，个人感觉也挺迷惑的。

**Comment by CloudyEagle** [comment]
也查看了播放器版本均没有极大的跳动，这B在灰测什么东西吗？
此脚本能否破除这个现象

**Comment by the1812** [comment]
不太清楚是为啥, 但是模拟 Safari 为什么能更好解码

**Reply by CloudyEagle** [reply]
我是听说可以更好地调度显卡解码视频，一顶程度上能减少对CPU解码的负担，可能具体也没啥卵用。
